app/views/app_settings/_form.html.erb

```diff
@@ -29,6 +29,22 @@
       </div>
     </div>
 
+    <div class="row border-bottom mb-4 pb-4">
+      <div class="col">
+        <h5>VM Settings</h5>
+      </div>
+      <div class="col-6">
+        <%= form.labeled_text_field 'Minimum cpu core count(GB)', :min_cpu_cores %>
```

Comment:
The GB here and in line 39 are copy & paste artifacts, right?

Reply:
Yes